kornia/color/lab.py

```diff
@@ -15,14 +15,16 @@
 def rgb_to_lab(image: torch.Tensor) -> torch.Tensor:
     r"""Converts a RGB image to Lab.
 
+    .. image:: _static/img/rgb_to_lab.png
```

Comment:
@cceyda @ducha-aiki @shijianjian @lferraz how should we represent this ? by default we get now something like this
![image](https://user-images.githubusercontent.com/5157099/123257548-99a78980-d4f2-11eb-8c20-d9d104195014.png)

Reply:
showing any image not encoded in RGB makes no sense...
So maybe we can show a single pixel value.  
rgb(before conversion) & lab (after conversion) input[0,:,0,0] vs output[0,:,0,0] 
something like this:
![image](https://user-images.githubusercontent.com/15624271/123260161-376d7980-d530-11eb-8a53-d6e3586fbe57.png)